murdock/murdock.py

```diff
@@ -582,7 +582,7 @@ async def handle_push_event(self, event: dict):
         ) or (
             ref_type == "tags" and not Murdock.handle_ref(ref_name, config.push.tags)
         ):
-            LOGGER.debug(f"Ref '{ref_name}' not accepted for push events")
+            LOGGER.info(f"Ref '{ref_name}' not accepted for push events")
```

Comment:
I think debug level is correct here.

Reply:
It's in the test that debug level should be enabled.

Reply:
You're right here, dropped the commit that changes this